keyboards/hhkb/keymaps/brett/keymap.c

```diff
@@ -0,0 +1,120 @@
+/*  -*-  eval: (turn-on-orgtbl); -*-
+ * Brettm12345 HHKB Layout
+ */
+#include QMK_KEYBOARD_H
+#include "brett.h"
+
+enum layers {
+   BASE = 0,
+   HHKB = 1,
+   PROG = 2,
+   MOUSE = 3
```

Comment:
```suggestion
   BASE,
   HHKB,
   PROG,
   MOUSE
```

Reply:
Specifically, you don't need to specify all of the numbers, since the enum handles this automatically.